index.html

```diff
@@ -42,47 +43,58 @@
 }
 </pre>
 
-## Dependencies
-The <dfn>`ReadableStream`</dfn> and <dfn>`WritableStream`</dfn> interfaces are
-defined in [[!STREAMS]].
+## Terminology and conventions
 
-The <dfn>`Promise`</dfn> and <dfn>`DOMException`</dfn> interfaces are defined
-in [[!DOM]].
+The terms
+<dfn data-cite="!STREAMS#rs-model">`ReadableStream`</dfn> and
+<dfn data-cite="!STREAMS#ws-model">`WritableStream`</dfn>
+are defined in [[!STREAMS]].
+
+The term
+<dfn data-cite="!ECMASCRIPT#sec-promise-objects">`Promise`</dfn>
+is defined in [[!ECMASCRIPT]].
+
+The term
+<dfn data-cite="!WEBIDL#idl-DOMException">`DOMException`</dfn>
+is defined in [[!WEBIDL]].
+
+The term <dfn data-cite="!HTML#eventhandler">EventHandler</dfn> is defined in [[!HTML]]
 
 <section data-dfn-for="Navigator" data-link-for="Navigator">
-<h2><dfn>Navigator</dfn> partial interface</h2>
+### <dfn>Navigator</dfn> partial interface
+
 <pre class="idl">
 [Exposed=Window, SecureContext]
 partial interface Navigator {
   [SameObject] readonly attribute Serial serial;
 };
 </pre>
-<section>
-<h2><dfn>serial</dfn> attribute</h2>
+
+### <dfn>serial</dfn> attribute
 
 When getting, the <a>serial</a> attribute always returns the same instance of
 the <a>Serial</a> object.
 </section>
-</section>
 
 <section data-dfn-for="WorkerNavigator" data-link-for="WorkerNavigator">
 <h2><dfn>WorkerNavigator</dfn> partial interface</h2>
+
 <pre class="idl">
 [Exposed=DedicatedWorker, SecureContext]
 partial interface WorkerNavigator {
   [SameObject] readonly attribute Serial serial;
 };
 </pre>
-<section>
-<h2><dfn>serial</dfn> attribute</h2>
+
+### <dfn>serial</dfn> attribute
 
 When getting, the <a>serial</a> attribute always returns the same instance of
 the <a>Serial</a> object.
 </section>
-</section>
 
 <section data-dfn-for="Serial" data-link-for="Serial">
 <h2><dfn>Serial</dfn> interface</h2>
+
```

Comment:
The line above can probably use ##

Reply:
That creates a new section, is that fine?

Reply:
Should be. I’ll take a look.

Reply:
Oh, a section inside a section. Oh, neat bug. Ok, yeah... that’s a proper bug

Reply:
To be clear, I meant section > section but excluding a heading. That would be a bug. Anyway, will check tomorrow 😴

Reply:
done for all h2

Reply:
Thanks for doing all this!